modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/edit_form_instance_record.jsp

```diff
@@ -77,9 +77,11 @@ renderResponse.setTitle(GetterUtil.get(title, LanguageUtil.get(request, "view-fo
 			</c:if>
 		</div>
 
-		<react:component
-			module="admin/js/index.es"
-			props="<%= ddmFormDisplayContext.getDDMFormReactData() %>"
-		/>
+		<div id=<%= ddmFormDisplayContext.getContainerId() %>>
```

Comment:
Hi @natocesarrego,
Just out of curiosity, why is it necessary to do this? 😛

Reply:
It's necessary, @carolmariaabb, because the [Pages React component](https://github.com/liferay/liferay-portal/blob/b30d0d32a13478154ff8a311cf10a3c1303e6f3d/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/components/Pages.es.js#L33), invoked by the [Forms React component](https://github.com/liferay/liferay-portal/blob/b30d0d32a13478154ff8a311cf10a3c1303e6f3d/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/containers/Form.es.js#L229-L246), demands the `containerId` to render a Form which isn't read only or on view mode. 😉

Reply:
Got it! Thanks 😊